### DISCUSSION TITLE
Feature request: downvoting

### DISCUSSION THREAD

**Discussion by mphasak** [community/community#22290] [New to GitHub]
<p>Hi, I read through your Studies in Community series and appreciate how you’re facing these issues.</p>
<p>I’m sure you’ve already considered this, but I can’t find a public feature request, so I’d like to request adding the ability to downvote / flag posts.</p>
<p>So many issue discussions are polluted by people rudely demanding things from maintainers, or otherwise creating unhelpful noise.  We have the emoji tags but I think it’d be useful to have an explicit set of downvote flags, and some vote threshold past which a comment would be hidden.</p>
<p>Thanks!</p>

**Comment by Wabri** [comment]
<p>Hi <a class="mention" href="/mphasak">@mphasak</a>,</p>
<p>It’s not the same things but you can use <strong>Report Inappropriate Content</strong> option that you can find nearby the post:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/1513c87166ac4d723c39970f0593d44ca2b18eca.png" data-download-href="/uploads/short-url/30spwEfUsSvhWWBJNbHuRdJzKvU.png?dl=1" title="mphashasfj.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181070084-103b881b-1f9b-4d69-aec2-8f477d513aab.png" alt="mphashasfj.png" data-base62-sha1="30spwEfUsSvhWWBJNbHuRdJzKvU" width="661" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/1513c87166ac4d723c39970f0593d44ca2b18eca_2_661x500.png, https://aws1.discourse-cdn.com/github/original/1X/1513c87166ac4d723c39970f0593d44ca2b18eca.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/1513c87166ac4d723c39970f0593d44ca2b18eca.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/1513c87166ac4d723c39970f0593d44ca2b18eca_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">mphashasfj.png</span><span class="informations">802×606 41.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>-Gabriele-</p>
<blockquote>
<p>Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great content! </p>
</blockquote>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/mphasak">@mphasak</a>,</p>
<p>Thanks for being here! Giving maintainers more granular permissions is top of mind for us, we have an ongoing issue with this feature request and I’m adding your comment to it. Thanks for taking the time to provide this feedback. </p>
<p>Best,</p>
<p>Andrea</p>

**Comment by mphasak** [comment]
<p>Hey thanks <a class="mention" href="/wabri">@wabri</a>.  This is cool.  I wasn’t super clear too, was thinking something like this in the issues discussions on <a href="http://github.com" rel="nofollow noopener">github.com</a> proper!</p>

**Comment by mphasak** [comment]
<p>Awesome to hear.  Thanks <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>!</p>

**Comment by lee-dohm** [comment]
<p>You can also report issues, pull requests, or comments on them on <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>. See the <a href="https://help.github.com/en/articles/reporting-abuse-or-spam" rel="nofollow noopener">help documentation on reporting</a> for more information.</p>
